library/src/main/java/com/google/android/exoplayer/text/webvtt/WebvttParser.java

```diff
@@ -46,6 +47,10 @@
   private static final Pattern WEBVTT_FILE_HEADER =
       Pattern.compile(WEBVTT_FILE_HEADER_STRING);
 
+  private static final String WEBVTT_COMMENT_BLOCK_STRING = "^NOTE((\\u0020|\u0009).*)?$";
```

Comment:
Accordingly to WebVTT spec:

A WebVTT comment block consists of the following components, in the given order:
 - The string "NOTE".
- Optionally, the following components, in the given order:
  Either:
   - A U+0020 SPACE character or U+0009 CHARACTER TABULATION (tab) character.
   - A WebVTT line terminator.

So:
 1.- "^NOTE" wouldn't be valid because a separation character (space, tab or new line) is required between NOTE and the comment.
 2.- "^NOTE\\s" strictly speaking is not valid because "\s" includes characters out of space and tab characters although, in the practice, I think it could be valid. It would be weird having someone using a vertical tab or a form feed (included by "\s") wrongly and unintentionally for separating NOTE keyword from the comment text.

Regarding point 2, and for consistency, we can do the same for WEBVTT regex:
 - From: "^\uFEFF?WEBVTT((\u0020|\u0009).*)?$"
 - To: "^\uFEFF?WEBVTT\\s"